### PR TITLE
Added sample code for MSBuild aliases

### PR DESCRIPTION
**Pull request by devlead**

### CODE REVIEW

File: src/Cake.Common/Tools/MSBuild/MSBuildAliases.cs [hunk 1]
```diff
@@ -28,6 +33,16 @@ public static void MSBuild(this ICakeContext context, FilePath solution)
         /// <param name="context">The context.</param>
         /// <param name="solution">The solution to build.</param>
         /// <param name="configurator">The settings configurator.</param>
+        /// <example>
+        /// <code>
+        /// MSBuild("./src/Cake.sln",
```
Comment (patriksvensson) [inline src/Cake.Common/Tools/MSBuild/MSBuildAliases.cs]: Suggestion for indentation:

```
MSBuild("./src/Cake.sln", configurator => 
  configurator.SetConfiguration("Debug")
    .SetVerbosity(Verbosity.Minimal)
    .UseToolVersion(MSBuildToolVersion.VS2015)
    .SetMSBuildPlatform(MSBuildPlatform.x86)
    .SetPlatformTarget(PlatformTarget.MSIL));
```

Comment (devlead) [inline src/Cake.Common/Tools/MSBuild/MSBuildAliases.cs]: Sure updated & rebased.

**Comment by patriksvensson** [comment]
@devlead Can you rebase this for merge?

**Comment by devlead** [comment]
@patriksvensson rebased.

**Comment by patriksvensson** [comment]
Related to #750.

**Pull request merged**